changelog: add 1.37.0 release notes (#4488)
bottlerocket-os/bottlerocket: CHANGELOG.md

# v1.37.0 (2025-04-21)

## Release Highlights
* Fix `aws-signing-helper` and IAM Roles Anywhere ([bottlerocket-core-kit#451](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/451))

> * Clear configuration-files and services in migrator ([bottlerocket-core-kit#456](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/456))

Reviewer: ```suggestion
* Clear configuration-files and services on upgrades ([bottlerocket-core-kit#456](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/456))
```

> * Replace `shimpei` and the `oci-add-hooks` packages with NVIDIA Container Runtime to handle OCI hook injection ([bottlerocket-core-kit#458](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/458))

Reviewer: ```suggestion
* Remove `shimpei` and `oci-add-hooks` ([bottlerocket-core-kit#458](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/458))
```

And then another item for:

```
* Use the NVIDIA Container Runtime to configure GPUs for containers
```

* Update `bottlerocket-kernel-kit` from 2.1.0 to 2.2.2 [CHANGELOG](https://github.com/bottlerocket-os/bottlerocket-kernel-kit/blob/develop/CHANGELOG.md#v222-2025-04-18) ([commits](https://github.com/bottlerocket-os/bottlerocket-kernel-kit/compare/v2.1.0...v2.2.2)) ([#4485], [#TODO])
* Update admin container from 0.11.18 to 0.11.19 [CHANGELOG](https://github.com/bottlerocket-os/bottlerocket-admin-container/blob/develop/CHANGELOG.md#01119) ([commits](https://github.com/bottlerocket-os/bottlerocket-admin-container/compare/v0.11.18...v0.11.19)) ([#4476])
* Update bootstrap container from 0.1.3 to 0.2.0 [CHANGELOG](https://github.com/bottlerocket-os/bottlerocket-bootstrap-container/blob/develop/CHANGELOG.md#020) ([commits](https://github.com/bottlerocket-os/bottlerocket-bootstrap-container/compare/v0.1.3...v0.2.0)) ([#4476])
* Update control container from 0.8.1 to 0.8.2 [CHANGELOG](https://github.com/bottlerocket-os/bottlerocket-control-container/blob/develop/CHANGELOG.md#082) ([commits](https://github.com/bottlerocket-os/bottlerocket-control-container/compare/v0.8.1...v0.8.2)) ([#4476])

## Build Changes
* Update Rust dependencies ([#4468], [#4469])

> * Update secure boot keys to include aws-partition & ca-signing-algo arguments. ([#4450])

Reviewer: nit: in some of your line items you use `.` at the end but not in all of them

Author: Thanks, missed one!
